packages/twenty-front/src/effect-components/PageChangeEffect.tsx

```diff
@@ -163,10 +167,14 @@ export const PageChangeEffect = () => {
 
   useEffect(() => {
     setTimeout(() => {
```

Comment:
Not sure what was the use-case for setting a timeout here? Do we want to use `useDebouncedCallback` maybe instead?